.gitignore

```diff
@@ -0,0 +1,2 @@
+/*build*
```

Comment:
Why the wildcards? Wouldn't be a simple /build/ enough?

Reply:
same as for the other PR